x-pack/plugins/console_extensions/spec/overrides/xpack.ml.get_overall_buckets.json

```diff
@@ -0,0 +1,13 @@
+{
+  "xpack.ml.get_overall_buckets": {
+    "data_autocomplete_rules": {
+      "allow_no_jobs": [true, false],
```

Comment:
This and `exclude_interim` only populate the field with an empty array, instead of `[true, false]`. `true` and `false` _are_ suggested by the autocompleter if you start typing "t" or "f" however.

Reply:
Should be fixed now, thanks for catching.